lib/index.js

```diff
@@ -24,10 +24,10 @@ class Oppsy extends Events.EventEmitter {
           psup: ProcessMonitor.uptime,
           psmem: ProcessMonitor.memoryUsage,
           psdelay: ProcessMonitor.delay,
-          requests: this._networkMonitor.requests,
-          concurrents: this._networkMonitor.concurrents,
-          responseTimes: this._networkMonitor.responseTimes,
-          sockets: this._networkMonitor.sockets
+          requests: this._networkMonitor.requests.bind(this._networkMonitor),
```

Comment:
:) i dunno what to name them, i just removed 's'
https://gist.github.com/bboysathish/c8d816961aa84bbcc6ff#file-index-js-L72
https://gist.github.com/bboysathish/c8d816961aa84bbcc6ff#file-index-js-L98

Do you have any suggestions?